irohad/subscription/scheduler_impl.hpp

```diff
@@ -49,7 +49,7 @@ namespace iroha::subscription {
     utils::WaitForSingleObject event_;
 
     /// Flag that shows if current handler is in task execution state
-    std::atomic<bool> is_busy_;
+    bool is_busy_;
```

Comment:
please separate this changes if they are not required for RDB

Reply:
This need to execute determine tests .